controllers/util/k8sutil.go

```diff
@@ -71,6 +71,7 @@ const (
 	CephRBDMirrorName                    = "cephrbdmirror"
 	OcsClientTimeout                     = 10 * time.Second
 	StorageClientMappingConfigName       = "storage-client-mapping"
+	ExternalClassLabelKey                = "storageclass.kubernetes.io/is-external"
```

Comment:
instead of kubernetes.io can we use ocs.openshift.io? maybe something like `ocs.openshift.io/external-ceph-class` or if you can think of something better

Reply:
how about storageclass.ocs.openshift.io/is-external?